Rust/tokio-echo-tcp/Enarx.toml

```diff
@@ -12,6 +12,6 @@ kind = "stderr"
 [[files]]
 name = "LISTEN"
 kind = "listen"
-port = 12345
+port = 10001
```

Comment:
Let's use `10010` here to have some room to add more

For example, it would probably be a very good idea to add a TLS port here by default. We could just listen on 10011 for that without having to look through all other examples or significantly changing config